<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>451</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="pageOne">
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>61</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_One">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_One_1">
        <item>
         <widget class="QLabel" name="labelWelcome">
          <property name="text">
           <string>Welcome Write Messege</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMessageInput">
          <property name="text">
           <string>Welcome Chat Server</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="veritalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>48</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="labelPort">
          <property name="text">
           <string>Selection Port Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxPort">
          <property name="minimum">
           <number>2000</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>2999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="pageTwo">
    <widget class="QLabel" name="lblStatusInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Bağlantı kesildi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartAction">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Başlat</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStopAction">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Durudur</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditServerMsg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
